app/views/layouts/_footer_meta_provider.html.erb

```diff
@@ -1,14 +1,39 @@
 <h2 class="govuk-heading-m"><%= t('layout.support.title') %></h2>
-<ul class="govuk-footer__meta-custom govuk-list govuk-!-font-size-16">
-  <li>Email: <%= bat_contact_mail_to(html_options: { class: 'govuk-footer__link' }) %></li>
-  <li><%= t('layout.support.response_time') %></li>
-  <li>Alternatively, <%= link_to 'give feedback through our survey', ProviderInterface::FEEDBACK_LINK, class: 'govuk-footer__link' %></li>
```

Comment:
We seem to have removed the survey link, we sure we don't want this anymore?

Reply:
Good question — @adamsilver / @johndoates?

Reply:
Yes, that was a conscious decision. We have the link in the header already. And I think we’re going to ask users to fill out the survey via email to increase response rate.

https://bat-design-history.netlify.app/manage-teacher-training-applications/helping-users-through-online-chat/#removal-of-survey-link

Reply:
Confirmed ok https://ukgovernmentdfe.slack.com/archives/CPH8J9G65/p1625561131174000